pkg/scheduler/factory/factory.go

```diff
@@ -1270,10 +1275,9 @@ func (i *podInformer) Lister() corelisters.PodLister {
 }
 
 // NewPodInformer creates a shared index informer that returns only non-terminal pods.
-func NewPodInformer(client clientset.Interface, resyncPeriod time.Duration, schedulerName string) coreinformers.PodInformer {
+func NewPodInformer(client clientset.Interface, resyncPeriod time.Duration) coreinformers.PodInformer {
 	selector := fields.ParseSelectorOrDie(
-		"spec.schedulerName=" + schedulerName +
-			",status.phase!=" + string(v1.PodSucceeded) +
+		"status.phase!=" + string(v1.PodSucceeded) +
```

Comment:
nit: we don-t need to check pod's phase again in `unassignedNonTerminatedPod` :)

Reply:
Yes, I thought about that too, but I decided not to change that part of the logic in this PR.